### ISSUE TITLE
Auto brief or """ not working for python

### ISSUE THREAD

**Issue by FelixKerger** [doxygen/doxygen#7953]
**Describe the bug**
From my understanding the auto brief feature should take the first line of a commend and use it as the brief. Looking at the example in the manual https://www.doxygen.nl/manual/docblocks.html#pythonblocks it seems the output linked there(https://www.doxygen.nl/manual/examples/docstring/html/classdocstring_1_1_py_class.html) is broken. No method, class or package has a brief text.

Reproducing it locally I get the same behaviour until I use `"""!` instead of `"""`, then it works as expected. So not sure if autobrief is broken or doxygen does not pick up pydoc's comments correctly.

**Expected behavior**
```
def func():
    """Documentation for a function.
 
    More details.
    """
    pass
```
This should produce a brief using the first line.

**To Reproduce**
Use the sample from the manual and build it locally.

**Version**
OSX and doxygen 1.8.19, also tested it with 1.8.11.

**Comment by albert-github** [comment]
The documentation states:
>For Python there is a standard way of documenting the code using so called documentation strings. Such strings
are stored in doc and can be retrieved at runtime. Doxygen will extract such comments and assume they have to
be represented in a preformatted way.

Thus the text is shown just as it is given, shown by the preformatted block. When one wants to have the documentation one has to use the `##` construct of the `"""!` construct.

**Comment by FelixKerger** [comment]
Thank you for the fast answer.
Then I might have misunderstood the provided example, I was assuming that it was expected to show the brief.

Based on your comment is it correct to assume that everything in a normal pydoc comment(using """) will be part of the detailed description and not used for brief etc.?

Are there any settings to change that behaviour?
Would a contribution which a setting be something that is acceptable if I provided it?

**Comment by albert-github** [comment]
Would be a bit a waste of time as I did beat you to it on February 7, 2020: Using Python docstrings (#7575)

**Comment by FelixKerger** [comment]
:) Awesome. Just one more question then.
`def func():
    """ Documentation for a function.

    More details.
    """`
With that setting, the first line should show up as the brief of the method? Or am I missing something.

**Comment by FelixKerger** [comment]
Awesome. Can't wait for it to be merged.

**Comment by albert-github** [comment]
Most likely yes.

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by FelixKerger** [comment]
Great work thanks.
I build master locally and how it behaves as I would expect it.
What is the release cycle of doxygen?

**Comment by TechnicalPirate** [comment]
Awesome! Will also be looking forward to this release :)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.20.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
